docs/platform/developing/flashing-rpi.md

```diff
@@ -0,0 +1,499 @@
+# Flashing an Image to RPI
+
+This topic describes how to flash Tizen on SD card with or without IoT Setup Manager and setting up Raspberry Pi 3 or 4.
+
+## Supported systems
+
+|Division	|Ubuntu	|Windows	|
+|:---:		|:---:	|:---:		|
+|Headless 32-bit|Yes	|No			|
+|Headed 32-bit|Yes	|Yes		|
+|Headed 64-bit|Yes	|No			|
+
+> [!NOTE]
+> In Windows, only EXT4 type file of rootfs.img is supported to flash rpm bundles. If you are using Headless 32-bit and Headed 64-bit image, RPMs cannot be installed through IOT Setup Manager because rootsfs.img is of BTRFS type file.
+
+## Prerequisites
+
+### Operating system
+
+IoT Setup Manager supports the following operating systems:
+
+-   Windows 7 and later (64-bit and 32-bit)
+-   Ubuntu 14.04 LTS and later (64-bit and 32-bit)
+
+### Ubuntu tools
+
+If you are using Ubuntu, the following packages must be installed in addition to the IoT Setup Manager:
+
+-   rpm2cpio
+-   cpio
+
+If the packages are not installed, run the following command to install a package:
+
+```
+sudo apt-get install <package-name>
+```
+
+### Download binaries
+
+You must have the supported binary images in your computer. To download the binary images, follow these steps:
+
+1. Download images from [Downloads](http://download.tizen.org/releases/milestone/tizen).
+2. Select **6.0-unified (or "unified" whichever is present) > latest > images > standard**.
+3. Download the compressed file for different devices or profiles from:
+
+     1.   RPI3 Headless 32-bit:
+          1.   Boot Image: iot-boot-armv7l-rpi3/
+          2.   Platform Image: iot-headless-2parts-armv7l-btrfs-rootfs-rpi/
+
+     2.   RPI3 Headed 32-bit:
+          1.   Boot Image: iot-boot-armv7l-rpi3/
+          2.   Platform Image: iot-headed-3parts-armv7l-rpi/
+
+     3.   RPI4 Headless 32-bit:
+          1.   Boot Image: iot-boot-armv7l-rpi4/
+          2.   Platform Image: iot-headless-2parts-armv7l-btrfs-rootfs-rpi/
+
+     4.   RPI4 Headed 32-bit:
+          1.   Boot Image: iot-boot-armv7l-rpi4/
+          2.   Platform Image: iot-headed-3parts-armv7l-rpi/
+
+     5.   RPI3 Headed 64-bit:
+          1.   Boot Image: iot-boot-arm64-rpi3/
+          2.   Platform Image: iot-headed-3parts-aarch64-rpi/
+
+     6.   RPI4 Headed 64-bit:
+          1.   Boot Image: iot-boot-arm64-rpi4/
+          2.   Platform Image: iot-headed-3parts-aarch64-rpi/
+
```

Comment:
Question. Are you going to remove "Environment" section, which is  about the JRE 1.8 or later?

Reply:
Now we are using the Java that comes with Tizen Studio to launch IoT Setup Manager. So there is no need for Java in the system environment. And therefore we removed the "Environment" section.

Reply:
To launch ISM, Java is not required to be installed in system environment as we are using Java that comes with Tizen Studio and not the system Java. So we removed "Environment" section. Please let us know if you think we should keep that section.

Reply:
Thanks for the reply. I just wanted to know the reason of removing. If so, we don't need to keep the section.